content/docs/ui/account-and-settings/troubleshooting-delays-and-latency.md

```diff
@@ -40,27 +40,52 @@ Experiencing delays? Check our [Status Page](http://status.sendgrid.com/) for th
-2. How to run a traceroute on _Mac_: Open “terminal” or “network utility” and type “traceroute smtp.sendgrid.net”
-3. How to run a traceroute on _Windows_: Open your command prompt, and type ”tracert smtp.sendgrid.net”.
-
-**As with the ping method, keep an eye out for any large times over 150.000 ms**. You'll also want to pay close attention to notice if there are any major increases from one hop to the next. This could indicate the inherent latency from when the packet leaves a server in France en route to a server in Canada. Another flag to watch out for are any of the increases early in the transit, as this could mean the latency lies within your local network, or at a certain ISP.
-
-Keep in mind that many modern network nodes will de-prioritize ICMP packets, which traceroute sends, so timeouts at certain hops may not necessarily be indicative of a faulty connection. That said, a traceroute can still go along way in helping isolate where a network issue exists.
-
-![]({{root_url}}/images/smtpTRACE.gif)
+**Hping - traceroute mode**
+
+1. A traceroute is along the same lines as a ping, you can think of a traceroute as a deeper analysis in the sense that it allows you to see at which “hop” the latency may begin to occur. You are able to see every stop along the route that the packet travels from the customer’s network to SendGrid’s servers. At each stop, details are given as to the address it travels to, and the amount of time to get there also in milliseconds.
+2. How to run a traceroute: Open “terminal” and type “sudo /usr/local/sbin/hping smtp.sendgrid.net -p 587 -S -T -c 10”
+```
+sudo /usr/local/sbin/hping smtp.sendgrid.net -p 587 -S -T -c 10
+HPING smtp.sendgrid.net (en0 169.45.113.201): S set, 40 headers + 0 data bytess
+hop=1 TTL 0 during transit from ip=172.22.16.1 name=UNKNOWN
+hop=1 hoprtt=2.8 ms
+hop=2 TTL 0 during transit from ip=4.31.56.1 name=UNKNOWN
+hop=2 hoprtt=15.9 ms
+hop=3 TTL 0 during transit from ip=4.69.203.121 name=ae-1-3501.ear3.sanjose1.level3.net
+hop=3 hoprtt=28.0 ms
+hop=4 TTL 0 during transit from ip=4.7.16.38 name=UNKNOWN
+hop=4 hoprtt=25.9 ms
+hop=5 TTL 0 during transit from ip=50.97.17.78 name=ae6.cbs02.eq01.sjc02.networklayer.com
+hop=5 hoprtt=25.2 ms
+```
+
+**As with the hping method, keep an eye out for any large times over 150.000 ms**. You'll also want to pay close attention to notice if there are any major increases from one hop to the next. This could indicate the inherent latency from when the packet leaves a server in France en route to a server in Canada. Another flag to watch out for are any of the increases early in the transit, as this could mean the latency lies within your local network, or at a certain ISP.

```

Comment:
Does specifying a latency value here imply that we promise a guarantee of latency?

Reply:
I think this just means that it'll be usually for a latency of 150ms per hop, not that we guarantee a 150ms latency

Reply:
150 ms is the latency from South Africa to our European location. That's as far away from our locations as our customers are likely to get. Values over 150 ms suggests they are being misrouted to the wrong location, or there's is some sort of congestion slowing traffic on the route.